R/testdata.R

```diff
@@ -55,91 +55,123 @@ dataset_test_worker <-
 
     # We're using 2nd edition of test that, which has "context" field
     # https://cran.r-project.org/web/packages/testthat/vignettes/third-edition.html
+    # Is there a reason to be using the 2nd edition, while context has been superseded in later versions?
```

Comment:
I fixed the edition as updating would require several tests to be updated and I had limited time available. You can update if you want. But it's also not necessary.

Reply:
Ah ok that's fine then! It works fine.